src/Delivery/DynamicEntry.php

```diff
@@ -88,51 +94,73 @@ public function __call($name, $arguments)
             $getId = true;
         }
 
-        if ($fieldConfig !== null && !$fieldConfig->isDisabled()) {
-            if (!isset($this->fields->$fieldName)) {
-                if ($fieldConfig->getType() === 'Array') {
-                    return [];
-                }
+        if ($fieldConfig === null || $fieldConfig->isDisabled()) {
+            trigger_error('Call to undefined method ' . __CLASS__ . '::' . $name . '()', E_USER_ERROR);
```

Comment:
`__call()` is the magic method invoked when the method called could not be found. The error message is emulating the one from the PHP engine when a method is missing, as we swallow that error by declaring `__call()`.